src/pages/components/input.mdx

```diff
@@ -131,8 +131,9 @@ import SizeCode from '!!raw-loader!../../examples/forms/inputs/Size.tsx';
 
 ### Textarea input
 
-A Textarea is for multiline text. By default, its height is fixed and becomes resizable with the
-`isResizable` prop.
+A Textarea is for multiline text. Textarea height is fixed by default but can be
+configured to resize automatically based on minimum and maximum rows or resize
```

Comment:
I'll lean on @vsrixyz more for this, but it may be cleaner to remove the mention of `minimum` and `maximum` rows. The height is determined by input, those values only define the bounds.

Something like:

> ... can be configured to resize automatically based on user input or manually with the `isResizable` prop

Reply:
I very much prefer the direction of your new line. I'll make that update in preparation for UXCS feedback.

Reply:
I like it. Made a few tweaks:

A Textarea is for multiline text. By default, Textarea height is fixed but can be configured to resize automatically based on user input or manually with `isResizable`.